.eslintrc.js

```diff
@@ -257,7 +260,9 @@ module.exports = {
 
       extends: [
         'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
```

Comment:
What's the difference in this one from the ones right before and right after?
It looks like it's already contained in the later line as well https://github.com/typescript-eslint/typescript-eslint/blob/a6f23a2dd5f1ad8e3ede0a4594a5e435182b4a2c/packages/eslint-plugin/src/configs/recommended-requiring-type-checking.ts#L6

Reply:
Good point, this is probably a mistake due to my various rebases. `plugin:@typescript-eslint/eslint-recommended` is not needed here, the other 2 are the recommended configuration [from their doc](https://typescript-eslint.io/linting/configs#recommended-requiring-type-checking)

Reply:
Interesting, looks like there is a "hard mode" too https://typescript-eslint.io/linting/configs/#strict